packages/react/src/CTAForm/CTAForm.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {forwardRef, type Ref} from 'react'
+import clsx from 'clsx'
+import {Button} from '../'
+
+import styles from './CTAForm.module.css'
+
+const Root = forwardRef(({...rest}: React.FormHTMLAttributes<HTMLFormElement>, ref: Ref<HTMLFormElement>) => {
+  return <form {...rest} ref={ref} className={clsx(styles.CTAForm, rest.className)} />
```

Comment:
nit: can we destructure the className in the fn arguments instead of using `rest.className`? Also moving the spread to the end?


```suggestion
const Root = forwardRef(({className, ...rest}: React.FormHTMLAttributes<HTMLFormElement>, ref: Ref<HTMLFormElement>) => {
  return <form ref={ref} className={clsx(styles.CTAForm, className)} {...rest}  />
```